src/Api/ApiClientBuilder.php

```diff
@@ -93,6 +93,7 @@ public function build()
         }
 
         $guzzleClient->setDefaultOption('exceptions', $this->httpExceptions);
+        $guzzleClient->setDefaultOption('verify',false);
```

Comment:
This option should be retrieved from the settings class and if it's empty, then default to `false`